src/azul/plugins/metadata/hca/transform.py

```diff
@@ -1010,7 +1010,10 @@ def _matrices(self, file: api.File) -> MutableJSON:
                 'version': file.manifest_entry.version,
                 'name': file.manifest_entry.name,
                 'size': file.manifest_entry.size,
+                # FIXME: Remove deprecated field 'matrix_cell_count'
+                #        https://github.com/DataBiosphere/azul/issues/3180
```

Comment:
Maybe add this on a separate commit.

Reply:
Good call. The point of doing so is that it creates an entry in the respective GH issue referencing the commit which in turn can be used to look at the FIXMEs which can be very helpful in understanding what the issue is all about.